src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1381,6 +1381,58 @@ def test_sql_elastic_pools_mgmt(self, resource_group, resource_group_location, s
                  checks=[NoneCheck()])
 
 
+class SqlElasticPoolOperationMgmtScenarioTest(ScenarioTest):
+    def __init__(self, method_name):
+        super(SqlElasticPoolOperationMgmtScenarioTest, self).__init__(method_name)
+        self.pool_name = "operationtestep1"
+
+    @ResourceGroupPreparer(location='southeastasia')
+    @SqlServerPreparer(location='southeastasia')
+    def test_sql_elastic_pool_operation_mgmt(self, resource_group, resource_group_location, server):
+        edition = 'Premium'
+        dtu = 125
+        db_dtu_min = 0
+        db_dtu_max = 50
+        storage = '50GB'
+        storage_mb = 51200
```

Comment:
This test would be much more readable if it used the kwarg-style syntax.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-5-get-more-from-resourcegrouppreparer

Reply:
@tjprescott agree. Will refine it in next PR. Since we are making breaking changes for database and pool web API now, would also needs to re-write the database and pool related tests.